cmd/validate/validateNavigation.go

```diff
@@ -15,6 +18,18 @@ func validateNavigation(cwd string) {
 	for _, file := range modulesFiles {
 		documentLoader := gojsonschema.NewReferenceLoader(fmt.Sprintf("file://%s", file))
 		result, err := gojsonschema.Validate(schemaLoader, documentLoader)
+		
+		var data map[string]interface {}
+		
+		fileContent, status := ioutil.ReadFile(file)
+		handleErr(status)
+
+		ok:= json.Unmarshal(fileContent, &data)
```

Comment:
We should panic if there is a marshaling error. It does mean there is something wrong with the format.